Merge initial environment details. (#2)
broadinstitute/cell-painting-batch-effects: README.md

# Examining Batch Effects on Cell Painting Data

This project examines the presence of batch effects in (high-throughput) single-cell morphology data.
We use [kBET](https://github.com/theislab/kBET) to first detect the extent of batch effects, and then [Harmony](https://github.com/immunogenomics/harmony) to adjust for them.

> Ultimately, our goal is to benchmark these two processes and evaluate their effectiveness single cell Cell Painting data.

Reviewer: ```suggestion
Ultimately, our goal is to benchmark these two processes and evaluate their effectiveness in single cell Cell Painting data.
```

# Examining Batch Effects on Cell Painting Data

> This project examines the presence of batch effects in (high-throughput) single-cell morphology data of different strains of cells subjected to various perturbations.

Reviewer: ```suggestion
This project examines the presence of batch effects in (high-throughput) single-cell morphology data.
```

> In addition to evaluating the benefits of the kBET detection algorithm, Harmony is used to correct for the observed batch effect. Ultimately, the goal is to benchmark these two processes and evaluate their effectiveness on the LINCS cell morphology data.

Reviewer: ```suggestion
We use [kBET](https://github.com/theislab/kBET) to first detect the extent of batch effects, and then [Harmony](https://github.com/immunogenomics/harmony) to adjust for them.
Ultimately, our goal is to benchmark these two processes and evaluate their effectiveness single cell Cell Painting data.
```

Looks great, just some minor tweaks. Note the one-sentence-per-line rule of thumb for markdown styling (it makes editing sentences so much easier, line-by-line.

> @TODO update README
> 

Reviewer: ```suggestion
```

No need for this, let's just delete

Author: Oops, I meant to delete that, just wanted to see is annotations were a thing in Github like in Java.

## Computational Environment

We use [conda](https://docs.conda.io/en/latest/) to manage the computational environment.

After installing conda, execute the following to install and navigate to the environment:

```bash
# First, install the conda environment
conda env create -f environment.yml

# Then, activate the environment and you're all set!

> conda activate myenv

Reviewer: we should name this something more specific. How about `cp-batch-effect`? Feel free to choose something else, it just shouldn't be nondescript.
